src/main/java/edu/cornell/kfs/module/purap/document/authorization/IWantDocumentAuthorizer.java

```diff
@@ -147,8 +147,13 @@ private boolean isDocumentAtConfirmationStep(Document document) {
     
     public boolean canViewContractTab(Document document, Person person) {
         WorkflowDocument workflowDocument = document.getDocumentHeader().getWorkflowDocument();
-        return (isInOrgHierarchyOrProcurementAssistantNode(document) && (isCurrentUserOrgReviewer(person) 
-                || isCurrentUserProcurementContractAssistant(person))) || workflowDocument.isFinal();
+
+        if (workflowDocument.isEnroute() || workflowDocument.isFinal()) {
```

Comment:
Based on the description of the JIra
I think this should include a permission check for " SSC or Purchasing Assistant reviewers" the status should be not workflowDocument.isFinal()